doc/config.rst

```diff
@@ -386,6 +387,22 @@ General configuration
 
    __ https://sourceforge.net/p/docutils/code/HEAD/tree/trunk/docutils/
 
+.. confval:: smartquotes_excludes
+
+   This is a ``set`` whose default is::
+
+     {'language': ['ja'], 'builder': ['man', 'text']}
+
+   It gives sufficient condition to ignore the :confval:`smart_quotes` setting
+   and deactivate the Smart Quotes transform.  The keys must be ``'builder'``
+   or names of configuration settings.  The values are lists.  Please note
+   that in case of invocation of ``make`` with multiple builder names, the
+   first one counts.  Also, when using ``sphinx-build`` in succession for
+   different builders, one needs to use ``-E`` option (e.g. in case of an
+   ``html`` build followed by a ``man`` build, or conversely.)
```

Comment:
ah sorry, I am not used to ``sphinx-build`` but as it creates by default a ``.doctrees`` individual to each target, there is no need for ``-E``. The problem arises when using the "make-mode" only. Will fix docs.